Resources/Prototypes/Entities/Objects/Materials/ore.yml

```diff
@@ -333,6 +333,46 @@
   - type: Stack
     count: 1
 
+- type: entity
+  parent: Coal
+  id: Coal5
+  suffix: Five
+  components:
+  - type: Stack
+    count: 5
+
+- type: entity
+  parent: Coal
+  id: Coal10
+  suffix: Ten
+  components:
+  - type: Stack
+    count: 10
+
+- type: entity
+  parent: Coal
+  id: Coal15
+  suffix: Fifteen
+  components:
+  - type: Stack
+    count: 15
+
+- type: entity
+  parent: Coal
+  id: Coal20
+  suffix: Twenty
+  components:
+  - type: Stack
+    count: 20
+
+- type: entity
+  parent: Coal
+  id: Coal25
+  suffix: Twentyfive
+  components:
+  - type: Stack
+    count: 25
```

Comment:
The max amount of coal should probably be less, I can imagine cargo getting way too much coal too easily. 10-15 max, I reckon?

Reply:
Sure.